Library/Homebrew/readall.rb

```diff
@@ -48,14 +49,29 @@ def valid_formulae?(formulae)
       !failed
     end
 
+    def valid_casks?(casks)
+      failed = false
+      casks.each do |file|
+        Cask::CaskLoader.load(file)
+      rescue Interrupt
+        raise
+      rescue Exception => e # rubocop:disable Lint/RescueException
+        onoe "Invalid cask: #{file}"
+        puts e
+        failed = true
+      end
+      !failed
```

Comment:
Maybe name it `success` instead so we don't need the negation here?

Reply:
I'll perform that change everywhere then